pages/common/nnn.md

```diff
@@ -0,0 +1,36 @@
+# nnn
+
+> Interactive terminal file manager and disk usage nalyzer.
+> More information: <https://github.com/jarun/nnn>.
+
+- Open the current directory (or specify one as the first argument):
+
+`nnn`
+
+- Open an existing bookmark:
```

Comment:
How do we create a bookmark in the first place ? Maybe this isn't that important compared to other options ? If so, then let us leave it out.

Reply:
Bookmarks are defined in the `NNN_BMS` environment variable, for example in your `.bashrc` you could add:

    export NNN_BMS='d:~/Documents;m:/home/user/Music;c:~/Downloads/Google Chrome'

Reply:
Hmm .. needs extra explanation. I think we can leave this one out.

Reply:
I think this example is ok to have, after all it's probably one of the most used and useful ones. I would suggest editing its description to:

    - Open an existing bookmark (defined in the `NNN_BMS` environment variable):

Reply:
Sure